### PR TITLE
T: stub only type inference

### PR DESCRIPTION
**Pull request by Undin**

### CODE REVIEW

File: src/test/kotlin/org/rust/lang/core/type/RsTypificationTestBase.kt [hunk 1]
```diff
@@ -5,14 +5,31 @@
...
         InlineFile(code)
+        check(description)
+    }
+
+    protected fun stubOnlyTypeInfer(@Language("Rust") code: String, description: String = "") {
+        val testProject = fileTreeFromText(code)
+            .createAndOpenFileWithCaretMarker()
+
+        checkAstNotLoaded(VirtualFileFilter { file ->
+            !file.path.endsWith(testProject.fileWithCaret)
```
Comment (matklad) [inline src/test/kotlin/org/rust/lang/core/type/RsTypificationTestBase.kt]: Have you verified that this assert really fires if type inferences touches AST?

Comment (Undin) [reply src/test/kotlin/org/rust/lang/core/type/RsTypificationTestBase.kt]: At least, my current implementation of const expr evaluation fails:)

Comment (matklad) [reply src/test/kotlin/org/rust/lang/core/type/RsTypificationTestBase.kt]: :)

**Comment by matklad** [comment]
bors r+

An excellent change @Undin !

**Pull request merged**